apps/frontend/src/utils/auth.ts

```diff
@@ -48,10 +48,8 @@ export function checkAuthLoader() {
 
   const decodedToken = jwtDecode(user.token);
 
-  const isTokenExpired = decodedToken.exp ? Date.now() >= decodedToken.exp * 1000 : null;
-  console.log(isTokenExpired);
-  
-  // if (isTokenExpired) return redirect('/login');
+  const isTokenExpired = decodedToken.exp ? Date.now() >= decodedToken.exp * 1000 : null;  
```

Comment:
I think you haven't merged the latest changes to this branch. Main already returns a redirect to /login